Dockerfile

```diff
@@ -58,7 +58,7 @@ ENV APP_NAME=${APP_NAME} \
 # Update kernel and install runtime dependencies
 RUN apk --no-cache update && \
     apk --no-cache upgrade && \
-    apk --no-cache add ncurses-libs openssl bash erlang-crypto libssl1.1 yaml-dev nodejs
+    apk --no-cache add bash openssl erlang-crypto nodejs yaml-dev
```

Comment:
We need `nodejs` at runtime?

Reply:
Since the Emberjs build _has_ to be executed in the entrypoint for it to be twelve-factor, we need NodeJS.

Reply:
Ah right, this is a special case 😄